### PR TITLE
update deps and switch from ruby-sass to node-sass

### PR DESCRIPTION
**Pull request by phra**
simple maintenance:
- update deps
- switch from ruby-sass to node-sass

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -24,10 +24,8 @@ gulp.task('js', function() {
 
 // styles task
 gulp.task('styles', function() {
-  return sass('./sass', {
-      style: 'expanded',
-      noCache: true
-    })
+  return gulp.src('./sass/*.scss')
+    .pipe(sass().on('error', sass.logError))
```
Comment (callmenick) [inline gulpfile.js]: Add output style expanded here please.

File: package.json [hunk 2]
```diff
@@ -3,12 +3,14 @@
   "version": "0.1.0",
   "devDependencies": {
     "gulp": "^3.8.11",
-    "gulp-autoprefixer": "^2.1.0",
+    "gulp-autoprefixer": "^3.1.0",
     "gulp-concat": "^2.5.2",
     "gulp-minify-css": "^1.0.0",
     "gulp-rename": "^1.2.0",
-    "gulp-ruby-sass": "^1.0.1",
+    "gulp-ruby-sass": "^2.0.6",
```
Comment (callmenick) [inline package.json]: you should remove ruby sass as a dependency if it's no longer in use.

**Comment by phra** [comment]
fixed.

**Comment by callmenick** [comment]
you didn't recompile the sass though.

**Comment by phra** [comment]
@callmenick sorry, doing it now. also do i have to bump the version up? (e.g. 0.1.1)

**Comment by phra** [comment]
done.
i've also bump the minor version.
let me know if i have to drop 4cc0794

**Comment by phra** [comment]
any updates?

**Comment by phra** [comment]
any updates?

**Comment by phra** [comment]
@callmenick can you please at least answer me? :confused:

**Comment by phra** [comment]
i see conflicts on this PR. do you want me to fix them? i can also switch to [postcss](https://github.com/postcss/postcss) + [precss](https://github.com/jonathantneal/precss) to eliminate completely sass.

let me know what i have to do.

**Comment by callmenick** [comment]
@phra hey sorry there, I'm now going through the thread of comments from over 2 years ago... I've never seen them up until now 😞 didn't mean to leave you hanging.

**Pull request closed without merging**